aiida/orm/implementation/django/querybuilder.py

```diff
@@ -206,11 +206,11 @@ def get_filter_expr(self, operator, value, attr_key, is_attribute, alias=None, c
                     ''.format(operator, value)
                 )
         elif operator == 'in':
+            if not value:
+                raise InputValidationError('Value for operator \'in\' is an empty list')
```

Comment:
We indeed prefer single quotes for string literals but not when that requires escaping internal quotes. In this case favor 
```
'Value for operator `in` is an empty list'
```
or double quotes
```
'Value for operator "in" is an empty list'
```
or if you really want single quotes

```
"Value for operator 'in' is an empty list"
```